### ISSUE TITLE
2-5 second delay to load all pages

### ISSUE THREAD

**Issue by officiallymarky** [brave/brave-browser#15220]
## Description 

Loading any page in Brave or even submitting messages (aka on Reddit) results in a 2-5 second delay before the action starts.

## Steps to Reproduce

   1. Click link
   2. Wait 2-5 second
   3. Page starts to load

## Actual result:

2-5 second delay everywhere when using current build of Brave

## Expected result:

Click link
link loads

fill form
hit submit
it submits right away

## Reproduces how often: 

Happens 80%-90% of the time clicking on links, filling in forms, or doing anything.  I reinstalled OS to new drive, and it started back upup after about a week.

## Desktop Brave version:

1.22.71

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? No
- Does the issue resolve itself when disabling Brave Rewards? No
- Is the issue reproducible on the latest version of Chrome? No, no delays what so ever.

## Miscellaneous Information:

OS: Windows 10 Pro x64
Machine: Intel i7 4770K 32GB Ram 1GBit Internet
NVME Drives

I run only 1Password, Pocket, and Hive Keychain extension.  But it happens with or without extensions.

The delay is frequently paired with "waiting for ...", "estabilishing secure connection", "requesting..." in the bottom left.  I have used Fiddler to sniff traffic, and I don't see anything happening during these few seconds, just waiting.  I am using 1.1.1.1 with for my DNS with uses encrypted dns.  I have tried changing to 8.8.8.8 to see if it was related to that.

This problem started a couple of weeks ago, I put another NVME drive in my system and did a fresh install of Windows in parallel.  Problem did not exist for about a week, and now is showing the same symptoms.  I did see some relief for a short time after a fresh install, everything loaded immediately, but now it is unusable as I spend most of my days on the web working.

I can see it in other areas, not just loading pages.  For example, loading a YouTube video I get a spinning wheel for 2-5 seconds, the page is loaded but the video is delayed.  Leaving a comment on Reddit will say "submitting..." for 2-5 seconds.  It doesn't happen every click or page load, but it is around 80% of the time making trying to get anything done unbearable.

For example, just submitting this issue took around 4 seconds delay with "connecting..." in the bottom left, but nothing else happening, then all of a sudden it is instantly finished.

I have tried disabling all extensions, 
I have tried using a private browser window,
I have tried installing (fresh Windows install) Brave with very little other software installed.  
I tried manually downloading the next build of Brave 
I've tried nightly build

Sorry, I didn't remove the android section and just put N/A but that added Android label and I can't figure out how to remove it, this is not related to Android in any way.

**Comment by ryanbr** [comment]
I would suspect is something else causing it rather than the browser, A few things to check.

1. Are you using a VPN?  (either on the router/computer). If so disable.
2. Are you using Wifi or Ethernet?
3. Try pinging the router, and `1.1.1.1`. See what latiency results you get

**Comment by officiallymarky** [comment]
1) No, but enabling a VPN seems to minimize if not completely eliminate the issue.  This is something I just discovered, it isn't even a high end VPN.  
2) Ethernet
3) very consistent ~20ms, similar to 8.8.8.8, although I have noticed packet loss with 1.1.11 in the past (known issue many others have observed as well).

**Comment by rebron** [comment]
@officiallymarky Can you toggle brave://settings/system hardware acceleration and see if that's causing the problem.

**Comment by officiallymarky** [comment]
@rebron I did try that and didn't see any improvement.  I also watched Brave's task manager for anything that would stand out, but nothing did.

I think I might be on to the problem, doing some testing now.

I turned on these two block lists (the only two I had on) about the time the problem started happening, and it carried over to 2 full OS reinstalls, and one on completely new hardware.  I'm not 100% sure it solved the problem as I am skeptical of every slow down now, but I should know shortly after some more use.

![](https://i.imgur.com/BMv36ZC.png)

**Comment by ryanbr** [comment]
Just enable Fanboy Annoyances, since Easylist Cookie is apart of it

**Comment by officiallymarky** [comment]
Even with both disabled, I'm still getting these delays.
Oddly connecting to a VPN mostly eliminates the delays.

**Comment by GH-Rake** [comment]
I am also getting similar delay since updating 3 days ago

**Comment by officiallymarky** [comment]
Been using Firefox for almost two weeks now, not a single occurrence of a similar delay outside of a couple know slow sites. 

Until there is a fix, I am unable to use Brave on a day to day basis. 

I would be happy to try anything if you have ideas.

**Comment by GH-Rake** [comment]
Brave updated yesterday to version:
Version 1.23.73 Chromium: 90.0.4430.85 (Official Build) (64-bit)
(Windows 10 2004 19041.867)

But it's still having the same delay issue, so the problem is in both this version and the prior.

**Comment by pictosun** [comment]
Hi, 
same for me with newest Update on macOS (Version 1.23.75 Chromium: 90.0.4430.93 (arm64)).

I don't have those delay when using "private mode" within brave. (Don't use any plugins, deleted all caches, ...).

What I found out is, that the issues also occur with latest MS Edge Browser (same Chromium version).

Maybe it related to a bug within chromium?

**Comment by officiallymarky** [comment]
I recently gave Brave a try again, been using Firefox (very successfully) since this problem got so bad I couldn't work.
I am now using Version 1.25.68 Chromium: 91.0.4472.77 (Official Build) (64-bit) and it feels like the problem has mostly been resolved.  It is still slower than Firefox and it happens occasionally, but it hasn't been every other page like it was previously.  It is likely an upstream problem with Chromium, but I did not see any issues in Chrome, so I can't confirm that.

Either way, it isn't as snappy as it was before on a very fast (1Gbit) connection and fast computer (5950X) but it isn't a show stopper anymore.
